### ISSUE TITLE
Very slow typescript autocomplete/intellisense

### ISSUE THREAD

**Issue by p32929** [microsoft/vscode#129935]
Does this issue occur when all extensions are disabled?: Yes

Steps to Reproduce:

1. Open any NodeJS-TypeScript project
2. Write the first letter of a class or variable
3. Wait for the intellisense to show

Here's a screen record of the problem:
![Vs](https://user-images.githubusercontent.com/6418354/127800226-3ce84aad-a5c8-4726-814f-7399c2ce77e8.gif)

Version: 1.58.2 (user setup)
Commit: c3f126316369cd610563c75b1b1725e0679adfb3
Date: 2021-07-14T22:10:15.214Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.18363
Typescript version: 4.3.5

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by p32929** [comment]
Yes, even after disabling all the extensions, in the latest VScode

**Comment by outranker** [comment]
@mjbvz you didn't request any info to further investigate the issue but labelled it as needs more info. is issue creator responsible for providing additional details about the performance problem?

**Comment by p32929** [comment]
Is there any solution to this issue?

**Comment by p32929** [comment]
I will share the logs tomorrow

**Comment by p32929** [comment]
[tsserver.log](https://github.com/microsoft/vscode/files/6977111/tsserver.log)
Here's the file. Hope it gets fixed soon. Thanks
